arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -1,339 +1,492 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "VaultExtendedInfo"
+        ],
+        "description": "Create vault extended info.",
+        "operationId": "VaultExtendedInfo_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceResourceExtendedInfoDetails",
+            "in": "body",
+            "description": "resourceResourceExtendedInfoDetails",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "VaultExtendedInfo"
+        ],
+        "description": "Update vault extended info.",
+        "operationId": "VaultExtendedInfo_Patch",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "name": "resourceResourceExtendedInfoDetails",
+            "in": "body",
+            "description": "resourceResourceExtendedInfoDetails",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VaultExtendedInfoResource"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "Object": {

```

Comment:
Per @olydis comments remove this definition even if it is autogenerated by swashbuckle